packages/messaging/src/models/iid-model.ts

```diff
@@ -42,7 +43,12 @@ export class IidModel {
       `encryption_key=${p256dh}&` +
       `encryption_auth=${auth}`;
 
-    if (publicVapidKey !== DEFAULT_PUBLIC_VAPID_KEY) {
+    if (
```

Comment:
This formatting looks like the formatter tried to smear your PR over as many lines as possible, turning it into some kind of weird symmetric ASCII-art :/